config/settings.yml

```diff
@@ -61,4 +61,6 @@ use_custom_pdf_viewer: true
 use_custom_3d_viewer: true
 collections_to_show_attribution:
   - dc876nn4469
-analytics_id:
+# Google Analytics will report in debug mode by default
+# in shared_configs for embed-prod, we set this value to 0  
+analytics_debug: 1
```

Comment:
Can we keep the default as 0, and set it to 1 where we need to debug?

Reply:
@jcoyne I guess the logic here is a "safety" style logic -- we don't want to mistakenly send data to our real analytics, so better to default to debug mode and explicitly set production.